locales/en/update-director-details.json

```diff
@@ -0,0 +1,18 @@
+{
+    "updateDirectorDetailsTitle" : "View and update the director's details",
+    "updateDirectorDetailsHint" : "If you need to tell us about more than one update, and they happened on different dates, you must submit them separately.",
+    "updateDirectorDetailsInsetText" : "Any updates you make to a director's details are pending until you’ve submitted this filing, and we’ve accepted it.",
+    "updateDirectorDetailsHeading" : "Director details",
+    "updateDirectorDetailsUpdate" : "Update",
+    "updateDirectorDetailsName" : "Name",
+    "updateDirectorDetailsNationality" : "Nationality",
+    "updateDirectorDetailsOccupation" : "Occupation",
+    "updateDirectorDetailsContinueButton" : "Continue",
```

Comment:
The file furniture.json contains commonly used strings such as "Continue"

Reply:
Thanks, have updated this now